### ISSUE TITLE
how can just use long-term relevance score?

### ISSUE THREAD

**Issue by bahareharandizade** [deezer/sigir23-mojito#5]
Hello, thanks for sharing this amazing code, I learned a lot by reviewing it. I have question about about lambda_glob, while in the paper you defined L = lambda * (L_short) + (1-lambda) *(L_long), in the code in _create_loss() function in models.net, you condition on lambda > 0, but at the end you defined loss as :
self.loss = loc_loss + self.lambda_glob * glob_loss

shouldn't be :

self.loss = (1-self.lambda_glob) * loc_loss + self.lambda_glob * glob_loss
instead?

In other words, if I just want to consider relevant score from long-term, and completely ignore short-term relevance score, how can I change this?

Many thanks for your time and this amazing work
